docs/source/api/stats_summary.rst

```diff
@@ -173,4 +176,76 @@ Last Updated Summary Stat
 
 	{ "response": {
 		"summaryTime": "2019-11-19 00:04:06+00"
-	}}
\ No newline at end of file
+	}}
+
+``POST``
+========
+
+.. versionadded:: 1.5
+
+Post a stats summary for a given stat.
+
+:Auth. Required: Yes
+:Roles Required: None
+:Response Type: Object
+
+Request Structure
+-----------------
+:cdnName:             The CDN name for which the summary stat was taken for
+
+	.. note:: If the ``cdn`` is equal to ``all`` it represents summary_stats across all delivery services within the given CDN
```

Comment:
This still says that `"cdn": "all"` means that the stats are for "... all delivery services within the given CDN"

Reply:
🤦‍♂